common/lib/xmodule/xmodule/js/src/video/09_completion.js

```diff
@@ -0,0 +1,142 @@
+(function(define) {
```

Comment:
Seems like there should be tests added to:
https://github.com/edx/edx-platform/tree/master/common/lib/xmodule/xmodule/js/spec/video

Reply:
Does thumbs-up mean this is coming?

Reply:
Yes.  I worked on tests on Friday, but wasn't able to get them all in place. I'll ping you again when they are ready or by end of day at the latest.